R/fimsframe.R

```diff
@@ -627,12 +627,13 @@ FIMSFrame <- function(data) {
   # Get the range of ages displayed in the data to use to specify population
   # simulation range
   if ("age" %in% colnames(data)) {
-    ages <- min(data[["age"]], na.rm = TRUE):max(data[["age"]], na.rm = TRUE)
+    ages <- sort(unique(data[["age"]]))
+    ages <- ages[!is.na(ages)]
```

Comment:
Does FIMS require equal size age bins? I worry with this change that there is no check to see there are data in annual age increments. Also, we should resort from creating an object, i.e., ages, and then modifying it and calling it ages again.

Reply:
I switched the code back to the original version. We had changed it earlier to match the length code when we ran into an error during testing.